### ISSUE TITLE
Test aria-live interruption

### ISSUE THREAD

**Issue by adamsilver** [adamsilver/f#122]
A: The container has a role attribute set to status and aria-live set to polite. When the status box is updated, screen readers will announce it, but only after the user finishes typing. This way, users aren't rudely interrupted.
H: Reminder that we need to test this. I think some screen reader / browser combos will interrupt, so there may be issues that might need addressing with debouncing or just activating the live region when the number of remaining characters becomes critical.

**Comment by adamsilver** [comment]
Won't fix.
